tasks/libs/pipeline_data.py

```diff
@@ -130,3 +130,19 @@ def truncate_job_name(job_name, max_char_per_job=48):
     # We also want to avoid it being too long
     truncated_job_name = truncated_job_name[:max_char_per_job]
     return truncated_job_name
+
+
+# those jobs are `allow_failure: true` but still needs to be included
+# in fail reports
+jobs_allowed_to_fails_but_need_report = [re.compile(r'kitchen_test_security_agent.*')]
+
+
+def should_include_failed_job(job_name, allow_failure):
```

Comment:
```suggestion
def should_report_job(job_name, allow_failure):
```
just a suggestion, sounds more explicit to me this way

Reply:
I can do this yes